var/spack/repos/builtin/packages/py-ase/package.py

```diff
@@ -14,8 +14,11 @@ class PyAse(PythonPackage):
     homepage = "https://wiki.fysik.dtu.dk/ase/"
     url      = "https://pypi.io/packages/source/a/ase/ase-3.13.0.tar.gz"
 
+    version('3.18.0',
+            sha256='39d45f12def2669605bffc82926acfb13a0d0610e6d82740fa316aafa70f97f9')
     version('3.15.0', '65a0143753517c2df157e53bd29a18e3')
     version('3.13.0', 'e946a0addc5b61e5e2e75857e0f99b89')
 
-    depends_on('python@2.6:')
+    depends_on('python@2.6:', when='@:3.15.0')
+    depends_on('python@3.5:', when='@3.18.0:')
```

Comment:
I would add `type=('build', 'run')` to these.

Reply:
Done.